src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Testing/MsTestV2/TestMethodRunnerExecuteIntegration.cs

```diff
@@ -54,8 +59,7 @@ public static CallTargetState OnMethodBegin<TTarget>(TTarget instance)
             string testSuite = testMethodInfo.TestClassName;
             string testName = testMethodInfo.TestMethodName;
 
-            Tracer tracer = Tracer.Instance;
-            Scope scope = tracer.StartActive("mstest.test");
+            Scope scope = TestTracer.StartActive("mstest.test", serviceName: ServiceName);
```

Comment:
To use the same instance consistently, we should probably replace all uses of `Tracer.Instance` with `TestTracer`, like line 49 above.

Reply:
thanks for the catch, I did a refactor to use the same instance in multiple classes in aac0b03